pandas/tests/io/pytables/test_select.py

```diff
@@ -956,7 +956,7 @@ def test_query_compare_column_type(setup_path):
                 with pytest.raises(ValueError, match=msg):
                     store.select("test", where=query)
 
-            for v, col in zip(
+            for v, col in zip(  # noqa: B007
```

Comment:
I don't understand why `v` cannot be removed here.

Reply:
best guess is because somewhere down the line the string on L962 is being passed to `eval` or `exec`